### PR TITLE
Update CLI to recover expired client for SDK 46 updates to gov module

### PR DESCRIPTION
**Pull request by tmsdkeys**
## Description

Update [docs on gov proposals](https://ibc.cosmos.network/main/ibc/proposals.html) (to recover frozen light clients) to include SDK v0.46 changes to x/gov API: https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADING.md#xgov-1

closes: #2173 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [X] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [X] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [X] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/ibc/proposals.md [hunk 1]
```diff
@@ -72,18 +76,34 @@ The client is attached to the expected Akash `chain-id`. Note that although the
...
 
-```
-<binary> tx gov submit-proposal update-client <expired-client-id> <active-client-id>
-```
+> Note that the Cosmos SDK has updated how governance proposals are submitted in SDK v0.46, now requiring to pass a .json proposal file
+
+- Starting SDK v0.46.x
+
+  ```
+  <binary> tx gov submit-proposal [path-to-proposal-json]
```
Comment (tmsdkeys) [inline docs/ibc/proposals.md:86]: We should probably add the `proposal.json` right?

File: docs/ibc/proposals.md [hunk 2]
```diff
@@ -72,18 +76,34 @@ The client is attached to the expected Akash `chain-id`. Note that although the
...
+
+  ```
+  <binary> tx gov submit-legacy-proposal update-client <expired-client-id> <active-client-id>
+  ```
+
+- Until SDK v0.45.x
+  ```
+  <binary> tx gov submit-proposal update-client <expired-client-id> <active-client-id>
+  ```
+  The `<expired-client-id>` identifier is the proposed client to be updated. This client must be either frozen or expired.
```
Comment (tmsdkeys) [inline docs/ibc/proposals.md:99]: Alignment needs to be fixed

File: docs/ibc/proposals.md [hunk 3]
```diff
@@ -72,18 +76,53 @@ The client is attached to the expected Akash `chain-id`. Note that although the
...
+        "@type": "/ibc.core.client.v1.ClientUpdateProposal",
+        "title": "title_string",
+        "description": "description_string",
+        "subject_client_id": "expired_client_id_string",
+        "substitute_client_id": "active_client_id_string"
+      }
+    ],
+    "metadata": "<metadata>",
+    "deposit": "10stake"
+  }
```
Comment (tmsdkeys) [inline docs/ibc/proposals.md:104]: Not sure if this is correct, if anyone can review..

**Review by colin-axner — APPROVED** [review]
LGTM! Thanks for updating this :pray: I haven't manually tested the json, but it looks correct to me

File: docs/ibc/proposals.md [hunk 4]
```diff
@@ -72,18 +76,53 @@ The client is attached to the expected Akash `chain-id`. Note that although the
...
+
+- Until SDK v0.45.x
+  ```
+  <binary> tx gov submit-proposal update-client <expired-client-id> <active-client-id>
+  ```
+
+The `<expired-client-id>` identifier is the proposed client to be updated. This client must be either frozen or expired.
 
-The `<active-client-id>` should be a client identifier on the same chain as the expired or frozen client. This client identifier should connect to the same chain as the expired or frozen client. This means: use the active client that is currently being used to relay packets between the two chains as the replacement client.
+The `<active-client-id>` represents a substitute client. It carries all the state for the client which may be updated. It must have identitical client and chain parameters to the client which may be updated (except for latest height, frozen height, and chain-id). It should be continually updated during the voting period.
```
Comment (crodriguezvega) [inline docs/ibc/proposals.md:120]: nit: 
```suggestion
The `<active-client-id>` represents a substitute client. It carries all the state for the client which may be updated. It must have identical client and chain parameters to the client which may be updated (except for latest height, frozen height, and chain ID). It should be continually updated during the voting period.
```

File: docs/ibc/proposals.md [hunk 5]
```diff
@@ -72,18 +76,53 @@ The client is attached to the expected Akash `chain-id`. Note that although the
...
 
-After this, it is just a question of who funds the governance deposit and if the chain in question votes yes.
+After this, all that remains is who funds the governance deposit and if governance propsosal passes. If it does, the client on trial will be updated to the latest state of the substitute.
```
Comment (crodriguezvega) [inline docs/ibc/proposals.md:122]:
```suggestion
After this, all that remains is deciding who funds the governance deposit and ensuring the governance proposal passes. If it does, the client on trial will be updated to the latest state of the substitute.
```

**Review by crodriguezvega — APPROVED** [review]
Thank you, @tmsdkeys!

**Pull request merged**